packages/block-library/src/navigation-link/link-ui.js

```diff
@@ -1,6 +1,7 @@
 /**
  * WordPress dependencies
  */
+import { __unstableStripHTML as stripHTML } from '@wordpress/dom';
```

Comment:
Using `stripHTML` was undefined. Had to use the unstable version. I don't understand how because it seems to be exported as stable...